docs/upgrade/xenial_upgrade_in_place.rst

```diff
@@ -0,0 +1,317 @@
+Ubuntu 16.04 LTS (Xenial) - Upgrading in Place
+==============================================
+
+The SecureDrop *Application* and *Monitor* servers may be upgraded-in-place to 
+Xenial using the procedure described here. This procedure does not require 
+physical access to the servers, but it will result in downtime for your instance
+- on the order of 8 hours if no issues are encountered, and longer if anything 
+goes wrong.
+
+The process is less complex than the alternative 
```

Comment:
This sentence is a little grammatically ambiguous. It is not clear what the preferred method actually is.

Reply:
hopefully clearer now!